client/my-sites/stats/features/modules/stats-emails/stats-emails.tsx

```diff
@@ -74,7 +81,25 @@ const StatsEmails: React.FC< StatsDefaultModuleProps > = ( {
 					}
 					additionalColumns={ {
 						header: <span>{ translate( 'Opens' ) }</span>,
-						body: ( item: { opens_rate: number } ) => <span>{ `${ item.opens_rate }%` }</span>,
+						body: ( item: EmailStatsItem ) => {
+							const opensUnique = parseInt( item.unique_opens, 10 );
+							const opens = parseInt( item.opens, 10 );
+							const hasUniques = hasUniqueMetrics( opensUnique, opens );
+							return (
+								<TooltipWrapper
+									value={
+										hasUniques
+											? `${ item.opens_rate }%`
+											: translate(
+													/* translators: Shown in a table column when email open rate data is not available */
+													'n/a'
```

Comment:
Wondering how this will get translated 🤔  Just as "—", and if it makes sense to directly use "—" instead?

Reply:
using "—" now 😄